pkg/ui/ccl/src/views/clusterviz/containers/map/circleLayout.tsx

```diff
@@ -63,7 +66,8 @@ export class CircleLayout extends React.Component<CircleLayoutProps> {
               <g transform={`translate(${this.coordsFor(i + childLocalities.length, total, radius)})`}>
                 <NodeView
                   node={node}
-                  liveness={this.props.liveness}
+                  livenessStatus={this.props.livenessStatus}
+                  liveness={this.props.liveness[node.desc.node_id]}
```

Comment:
it's super confusing that these props are inconsistent...

Reply:
Fixed them up to be consistent with the types — `livenessStatuses` is always `{ [id: string]: LivenessStatus }`; `livenesses` is `{ [id: string]: Liveness }`. Considered doing `livenessByNodeID` and `livnessStatusByNodeID`, but those are a little verbose.

I wish there was a single object that contained both, but that's not how the protobufs are organized, and I didn't feel like introducing a new type in TS…

Reply:
Yes, the Right Thing to Do (TM) is put them both on one thing, but probably not in the 2.0 timeframe.